libs/ts-rest/express/src/lib/types.ts

```diff
@@ -32,10 +38,21 @@ export type AppRouteImplementation<T extends AppRoute> =
     ? AppRouteQueryImplementation<T>
     : never;
 
-export type TsRestRequest<T extends AppRouter | AppRoute> =
-  Express['request'] & {
-    tsRestRoute: FlattenAppRouter<T>;
-  };
+export type TsRestRequest<
+  T extends AppRouter | AppRoute,
+  F extends FlattenAppRouter<T> = FlattenAppRouter<T>,
```

Comment:
These could also be defined in the body of this definition with some type helpers or something if you do not prefer this.

Reply:
I think this is fine, no need to over abstract for now.